share/spice/snow/snow.js

```diff
@@ -57,198 +66,102 @@ function nrio (api_result) {
         color: '#fff'
     });
 
-    // TODO: Contact this guy and make sure he's cool with us using it:
-    // http://www.zachstronaut.com/posts/2009/12/21/happy-xmas-winternet.html
-    var ww = 0;
-    var wh = 0;
-    var maxw = 0;
-    var minw = 0;
-    var maxh = 0;
-    var textShadowSupport = true;
-    var xv = 0;
-    var snowflakes = ["\u2744", "\u2745", "\u2746"];
-    var prevTime;
-    var absMax = 200;
-    var flakeCount = 0;
-
-    function makeItSnow() {
-        var detectSize = function () {
-            ww = $dom.width();
-            wh = $dom.height();
-            maxw = ww + 300;
-            minw = -300;
-            maxh = wh + 300;
-        };
-
-        detectSize();
-
-        $(window).resize(detectSize);
-
-        if (!$dom.css('textShadow')) {
-            textShadowSupport = false;
-        }
+function makeItSnow() {
```

Comment:
Please indent this code so it's inline with the above. 

Also, if you've borrowed any of this, I think we should leave a comment to give credit where it's due :)